### PR TITLE
Add iTeleport Connect.app v6.0.0.2 Cask

### PR DESCRIPTION
**Pull request by Saklad5**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit —download {{cask_file}}` is error-free.
- [x] `brew cask style —fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/iteleport-connect.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'iteleport-connect' do
+  version '6.0.0.2'
+  sha256 '2a845a052dfa04ae8a5c26da918944082f4a3a43ba3804cb8a078a77ad9f3b4c'
+
+  url "http://www.iteleportmobile.com/download/iTeleport%20Installer.v#{version}.dmg"
+  appcast 'http://www.iteleportmobile.com/connect/mac/new',
+          checkpoint: 'e7df5301267b725512751f09ecd0a260586044e2b1330b4ac7ea3a8c8d5257ff'
+  name 'iTeleport Connect For Mac'
```
Comment (vitorgalvao) [inline Casks/iteleport-connect.rb]: Keep this one. Remove the other two. See [docs](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/name.md) as to why.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Review by vitorgalvao — APPROVED** [review]
Can be merged as is if Travis passes, if there are no changes after this approval.

File: Casks/iteleport-connect.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 '2a845a052dfa04ae8a5c26da918944082f4a3a43ba3804cb8a078a77ad9f3b4c'
+
+  url "http://www.iteleportmobile.com/download/iTeleport%20Installer.v#{version}.dmg"
+  appcast 'http://www.iteleportmobile.com/connect/mac/new',
+          checkpoint: 'e7df5301267b725512751f09ecd0a260586044e2b1330b4ac7ea3a8c8d5257ff'
+  name 'iTeleport Connect'
+  homepage 'http://www.iteleportmobile.com/connect'
+  license :gratis
+
+  auto_updates 'true'
```
Comment (vitorgalvao) [inline Casks/iteleport-connect.rb]: This should be a boolean, not a string (`auto_updates true`).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 4b310793dd093d537699352bb6d3742fabb9579f. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/iteleport-connect.rb (hunk 2 above)
Comment (Saklad5) [inline Casks/iteleport-connect.rb]: “brew style” and the rest didn’t change it.

Comment (vitorgalvao) [inline Casks/iteleport-connect.rb]: > “brew style” and the rest didn’t change it.

Doesn’t mean it shouldn’t be changed. That’s why we have manual reviews.

Comment (vitorgalvao) [inline Casks/iteleport-connect.rb]: Added a [report](https://github.com/caskroom/rubocop-cask/issues/34) on this.

**Comment by Saklad5** [comment]
Could we remove the “awaiting user reply” tag from this?

**Comment by vitorgalvao** [comment]
@Saklad5 Sure, if you want, but once an issue is closed, we don’t really care and usually leave the tags.

**Pull request closed without merging**